Makefile

```diff
@@ -1,4 +1,4 @@
-artifact_name       := lfp-pay-web
+artifact_name       := pps-pay-web
```

Comment:
Update all instances of pps-pay-web to penalty-payment-web per renaming.

Reply:
Changed from pps-pay-web to penalty-payment-web